content/en/dynamic_instrumentation/_index.md

```diff
@@ -66,50 +65,54 @@ To enable Dynamic Instrumentation on a service, select its runtime and follow th
 
 Dynamic Instrumentation can help you understand what your application is doing at runtime. By adding a Dynamic Instrumentation probe you are exporting additional data from your application, without the need to do any code change or redeployment.
 
-### Creating a snapshot probe
-
-A *snapshot probe* exports the context in which it was configured to Datadog. It captures class properties, method arguments and local variables. For more information, read [How Dynamic Instrumentation Works][11].
+### Creating a probe
 
-To create a snapshot probe:
+Both log and metric probes require the same initial setup:
 
 1. Go to the [Dynamic Instrumentation page][12].
-2. Click **Create Probe** in the top right, or click the three dot context menu on a service and select **Add a probe for this service**.
-3. Select **Snapshot** as the probe type.
-4. If not prefilled, choose a service from the list.
-5. If not prefilled, choose runtime, environment and version.
-6. If you set up Source Code Integration for the service, you can select the file where you want to set the probe.
-7. In the source code, select a line on which to set the probe.
+1. Click **Create Probe** in the top right, or click the three dot context menu on a service and select **Add a probe for this service**.
+1. If not prefilled, choose a service from the list.
+1. If not prefilled, choose runtime, environment and version.
+1. In the source code, select a class and method or a source file and line where to set the probe.
+  1. If you set up Source Code Integration for your service, when setting line probes you will receive autocomplete suggestions for the source file and you will see the source code of the selected file when choosing the line.
 
-### Creating a metric probe
+### Creating a log probe
 
-Metric probes emit metrics at a chosen location in your code. Use the Dynamic Instrumentation expression language to reference numeric values from the context, such as from a local variable or a class field. For more information, read [How Dynamic Instrumentation Works][11].
+A *log probe* will emit a log when it executes.
 
-To create a metric probe:
+If you enable `Capture method parameters and local variables` on the log probe, it will also capture method arguments, local variables, class fields, the call stack and any exceptions from the execution context and add them to the log event - you can see the captured values in the Datadog UI. Since capturing this data is a more performance intensive operation, it is only enabled on one instance of your service which matches the desired environment and version configured on the probe. Probes with capture are rate limited to execute once every second.
```

Comment:
```suggestion
If you enable `Capture method parameters and local variables` on the log probe, it also captures the following values from the execution context and adds them to the log event:
- method arguments
- local variables
- class fields
- the call stack
- exceptions 
You can see the captured values in the Datadog UI. 

Because capturing this data is performance-intensive, it is enabled on only one instance of your service that matches the probe's environment and version settings. Probes with capture enabled are rate limited to execute once per second.
```